lib/process-services-cloud/karma.conf.js

```diff
@@ -31,13 +31,20 @@ module.exports = function (config) {
         proxies: {
             '/assets/': '/base/lib/process-services-cloud/src/lib/assets/',
             '/resources/i18n/en.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
+            '/resources/i18n/en-US.json': '/base/lib/process-services-cloud/src/lib/mock/en.json',
             '/resources/i18n/fr.json': '/base/lib/process-services-cloud/src/lib/mock/fr.json',
             '/base/assets/': '/base/lib/process-services/assets/',
+            '/base/lib/core/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
+            '/base/lib/process-services-cloud/src/lib/assets/images/': '/base/lib/core/src/lib/assets/images/',
```

Comment:
why are we passing the core as process services cloud ?

Reply:
Line 38 requires content which is available only in core library - without it I have these errors:
```
[33m27 09 2022 10:06:55.891:WARN [web-server]: [39m404: /_karma_webpack_/base/lib/process-services-cloud/src/lib/assets/images/ft_ic_raster_image.svg
```